src/zenml/cli/server.py

```diff
@@ -178,12 +173,17 @@ def up(
     if ip_address is not None:
         config_attrs["ip_address"] = ip_address
 
+    from zenml.zen_server.deploy.deployment import ServerDeploymentConfig
```

Comment:
the linter isn't going to like these mid-block imports :)

Reply:
fixed it!